pages/overview.py

```diff
@@ -3,10 +3,14 @@
 import dash_bootstrap_components as dbc
 
 # import the visualization cards
-from .visualizations.overview.total_contributor_growth import gc_total_contributor_growth
+from .visualizations.overview.total_contributor_growth import (
+    gc_total_contributor_growth,
+)
 from .visualizations.overview.commits_over_time import gc_commits_over_time
 from .visualizations.overview.issues_over_time import gc_issues_over_time
-from .visualizations.overview.active_drifting_contributors import gc_active_drifting_contributors
+from .visualizations.overview.active_drifting_contributors import (
```

Comment:
why this change?

Reply:
black formatter didn't like the line length I suppose. I didn't intentionally change that.